### PR TITLE
Fix/fix

### PR DESCRIPTION
**Pull request by Shylock-Hg**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### What does this PR do?
Fit the library ssl configuration upgrading.
Need https://github.com/vesoft-inc/nebula/pull/3471 , seems don't support plaintext in ssl port again.

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: src/common/ssl/SSLConfig.cpp [hunk 1]
```diff
@@ -18,6 +18,7 @@ namespace nebula {
 std::shared_ptr<wangle::SSLContextConfig> sslContextConfig() {
   auto sslCfg = std::make_shared<wangle::SSLContextConfig>();
   sslCfg->addCertificate(FLAGS_cert_path, FLAGS_key_path, FLAGS_password_path);
+  sslCfg->clientVerification = folly::SSLContext::VerifyClientCertificate::DO_NOT_REQUEST;
```
Comment (Aiee) [inline src/common/ssl/SSLConfig.cpp:21]: Does this mean that the certificate from the client could be omitted?

**Review by Aiee — COMMENTED** [review]
Could we unify ca-signed and self-signeed as one?

**Review by HarrisChu — APPROVED** [review]

File: src/common/ssl/SSLConfig.cpp (hunk 1 above)
Comment (HarrisChu) [reply src/common/ssl/SSLConfig.cpp:21]: how about using "always" and failed if verification fails

Comment (Shylock-Hg) [reply src/common/ssl/SSLConfig.cpp:21]: Not, it just check CA of server (which is new feature after library upgrading), so I disable it to keep same with before.

**Review by Aiee — APPROVED** [review]
I still believe we should not distinguish ca-signed and self-signed cert, maybe address this in another PR.

**Review by Aiee — APPROVED** [review]

**Review by HarrisChu — APPROVED** [review]

**Comment by Shylock-Hg** [comment]
> I still believe we should not distinguish ca-signed and self-signed cert, maybe address this in another PR.

Could you detail this proposal?

**Pull request merged**